Sources/MapboxNavigation/NavigationViewLayout.swift

```diff
@@ -102,34 +102,41 @@ extension NavigationView {
         
         layoutConstraints.append(contentsOf: wayNameViewConstraints)
         
-        let speedLimitViewTopContraint = speedLimitView.topAnchor.constraint(equalTo: topBannerContainerView.bottomAnchor,
-                                                                             constant: 10)
-        let speedLimitViewWidthContraint = speedLimitView.widthAnchor.constraint(equalToConstant: FloatingButton.buttonSize.width)
-        let speedLimitViewHeightContraint = speedLimitView.heightAnchor.constraint(equalToConstant: FloatingButton.buttonSize.height)
-        
-        var speedLimitViewConstraints = [
-            speedLimitViewTopContraint,
-            speedLimitViewWidthContraint,
-            speedLimitViewHeightContraint
-        ]
-        
-        switch floatingButtonsPosition {
-        case .topLeading:
-            let speedLimitViewTrailingConstraint: NSLayoutConstraint
-            if UIApplication.shared.statusBarOrientation == .landscapeRight {
-                speedLimitViewTrailingConstraint = speedLimitView.trailingAnchor.constraint(equalTo: trailingAnchor,
-                                                                                            constant: -10)
-            } else {
-                speedLimitViewTrailingConstraint = speedLimitView.trailingAnchor.constraint(equalTo: safeTrailingAnchor,
-                                                                                            constant: -10)
-            }
+        let speedLimitViewConstraints: [NSLayoutConstraint]
+        if let speedLimitViewLayoutGuide = speedLimitViewLayoutGuide {
```

Comment:
`NavigationView.speedLimitViewLayoutGuide` allows `NavigationView` users to change default position of `SpeedLimitView`. I plan to make this property public in future.